src/mmw/mmw/settings/gwlfe_settings.py

```diff
@@ -385,3 +385,11 @@
     'AttenLossRatePath': 0,  # Attenuation: Loss Rate: Pathogens
     'StreamFlowVolAdj': 1,  # Streamflow Volume Adjustment Factor
 }
+
+GWLFE_CONFIG = {
+    'NumWeatherStations': 2,  # Number of weather stations to consider for each polygon # NOQA
+    'Livestock': ['dairy_cows', 'beef_cows', 'hogs', 'sheep', 'horses'],
+    'Poultry': ['broilers', 'layers', 'turkeys'],
+    'MonthDays': [31, 28, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31],
```

Comment:
They don't in the original source code. This is only used to calculate the average discharge in liters per square meter for each month:

```python
discharge = np.array([float(mg_d * days * LITERS_PER_MGAL) / area
                      for days in MONTHDAYS])
```

Since it is the average, and not for a specific month in a specific year, we should be okay. I could potentially change it to `28.2425` (the average number of days in February), but that's not what they do in the original source code.